LibertyDSNP/frequency
# Custom RPC: Handle check

# Goal
The goal of this PR is to implement a handle-checking custom RPC which applies the same validation as used by `claim_handle` extrinsic.  The purpose is to help Providers do a pre-flight check before submitting user signatures. This will help minimize round trips during the new user flow.

Closes #1284 

# Discussion
## To Verify:
Check out this branch and `make start`.

This should return false in the JSON RPC output:
```shell
curl -H "Content-Type: application/json" -d '{"id":1, "jsonrpc":"2.0", "method": "handles_validateHandle", "params": ["Bobbay😀😀"]}' http://localhost:9944/
```
This should return true in the JSON RPC output:
```shell
 curl -H "Content-Type: application/json" -d '{"id":1, "jsonrpc":"2.0", "method": "handles_validateHandle", "params": ["BobbyTables"]}' http://localhost:9944/
```

# Checklist
